### DISCUSSION TITLE
Example full installation for automation use.

### DISCUSSION THREAD

**Discussion by 0x004a** [varbhat/exatorrent#264] [Ideas and Feature Requests]
Plan to post here:
- install as a subfolder of a domain,
- scripts to add, delete, and list torrents,
- script to receive and act on data from "hookurl",
- set global max upload limit.

Reference:
- API https://github.com/varbhat/exatorrent/blob/73de1c7b7563bf42d69f34e7b20706a8eb62c2d5/docs/API.md
- Websocket client https://github.com/websockets/wscat
- Nginx https://github.com/varbhat/exatorrent/blob/73de1c7b7563bf42d69f34e7b20706a8eb62c2d5/docs/deploy.md

**Comment by mckaygerhard** [comment]
right now i will try with debian 12

On debian stable cos users (normal users not geeks) wants stability event fashion software

```
apt-get install build-essential

echo "deb [trusted=yes] https://deb.debian.org/debian/ $(lsb_release -cs)-backports main" > /etc/apt/sources.list.d/debianbackports.list

apt-get install -t $(lsb_release -cs)-backports golang

echo "deb [trusted=yes] https://deb.nodesource.com/node_18.x/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/nodejs.list

apt-get purge nodejs nom && apt-get install nodejs=18.17.1*
```

**the problem is that after a while got that documentation is well made but it lack of details.. i can improve . by example doe snot specify the golang version (force to be one not in stable distros.. only in fashioned geek ones)**

Winbuntu 20 buils without any problem.. 
Winbuntu 16 cannot build it.. 
Alpine 18 builds with no issues but i used a hacked version..
